tests/providers/microsoft/teams/hooks/test_teams_webhook.py

```diff
@@ -0,0 +1,90 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import json
+import unittest
+
+from airflow.models import Connection
+from airflow.providers.microsoft.teams.hooks.teams_webhook import TeamsWebhookHook
+from airflow.utils import db
+
+
+class TestTeamsWebhookHook(unittest.TestCase):
```

Comment:
Can you write this with `pytest` instead? For net-new unit tests, the preference is to use `pytest` over `unittest`.